### ISSUE TITLE
Deprecation warning for Node v6

### ISSUE THREAD

**Issue by SpaceK33z** [hexojs/hexo-generator-feed#36]
When running hexo with this package in Node 6.2, I'm getting this warning:

```
node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.

==== JS stack trace =========================================

Security context: 0x34871afc9e59 <JS Object>#0#
    1: .node [module.js:568] [pc=0x34aefd8a0184] (this=0x10d26bfe67b9 <an Object with map 0x1e493e818161>#1#,module=0x319fc44ab799 <a Module with map 0x1e493e818739>#2#,filename=0x319fc44ab709 <String[115]: /myblog/node_modules/hexo-generator-feed/node_modules/fsevents/build/Release/fse.node>)
    2: load [module.js:~449] [pc=0x34aefdb1a2f6] (this=0x319fc44ab799 <a Module with map 0x1e493e818739>#2#,filename=0x319fc44ab709 <String[115]: /myblog/node_modules/hexo-generator-feed/node_modules/fsevents/build/Release/fse.node>)
    3: tryModuleLoad(aka tryModuleLoad) [module.js:417] [pc=0x34aefd83921d] (this=0x34871af04189 <undefined>,module=0x319fc44ab799 <a Module with map 0x1e493e818739>#2#,filename=0x319fc44ab709 <String[115]: /myblog/node_modules/hexo-generator-feed/node_modules/fsevents/build/Release/fse 1: v8::Template::Set(v8::Local<v8::Name>, v8::Local<v8::Data>, v8::PropertyAttribute)
 2: fse::FSEvents::Initialize(v8::Local<v8::Object>)
 3: node::DLOpen(v8::FunctionCallbackInfo<v8::Value> const&)
 4: v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&))
 5: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::(anonymous namespace)::BuiltinArguments<(v8::internal::BuiltinExtraArguments)1>)
 6: v8::internal::Builtin_HandleApiCall(int, v8::internal::Object**, v8::internal::Isolate*)
 7: 0x34aefd70961b
(node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.
```

Note that everything seems to work correctly, but still it's not optimal to see this every time.

**Comment by dynastywind** [comment]
Same problem as mine. Hope it can be solved soon. It's really annoying to see such info.

**Comment by zerda** [comment]
This warning is come from [fsevents](https://github.com/strongloop/fsevents), and fixed in [1.0.12](https://github.com/strongloop/fsevents/issues/127).
Try to update [nunjucks](https://github.com/mozilla/nunjucks) dependency to latest version (I'm using 2.4.2), problem solved.

**Comment by SpaceK33z** [comment]
Interesting, it was already solved a while ago by updating nunjucks, but this hasn't been released yet.
